bigquery/samples/tests/test_create_job.py

```diff
@@ -19,6 +19,6 @@
 def test_create_job(capsys, client):
 
     query_job = create_job.create_job(client)
-    client.cancel_job(query_job.job_id, location="US")
+    client.cancel_job(query_job.job_id)
```

Comment:
Same here. I don't believe the cancel operation supports auto-detection. It defaults to `US`.

```suggestion
    client.cancel_job(query_job.job_id, location=query_job.location)
```